### ISSUE TITLE
Semi-supervised training code

### ISSUE THREAD

**Issue by thangngoc89** [UCSC-VLAA/SwinMM#5]
Hi, thank you for releasing your works. I saw Semi-supervised is mentioned in the paper and figure but I didn't see any actual numbers or training code for it. Can you please elaborate?

**Comment by HUANGLIZI** [comment]
Thanks for your attention on our work. As for the semi-supervised results, you can check Table 5 in the paper if you like. You can utilize the JSON files we provide at https://github.com/UCSC-VLAA/SwinMM/blob/master/WORD/dataset/ for the semi-supervised fine-tuning.

**Comment by thangngoc89** [comment]
@HUANGLIZI Thank you for answering. I have checked the code path you provided and it looked like you still uses the label while doing "unsupervised" training given by the fact that you share train transform with supervised mode. Which has RandCropByPosNegLabeld transformation. I'm trying to make this work without that augmentation and I will report later here.

**Comment by HUANGLIZI** [comment]
Yes, you should disenable the RandCropByPosNegLabeld transformation in the "unsupervised" training.

**Comment by thangngoc89** [comment]
@HUANGLIZI thanks. I've succesfully trained SwinMM on semi-supervised task by doing what you suggested. 
I replaced RandCropByPosNegLabeld with RandSpatialCropSamplesd for patch sampling and observed improvements on test set over fully supervised on limited labelled dataset.
